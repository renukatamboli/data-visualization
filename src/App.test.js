import { render, screen } from "@testing-library/react";
import App from "./App";

test("match snapshot", () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});

test("renders home page", () => {
  render(<App />);
  const navElement = screen.getByTestId("nav");
  const homeElement = screen.getByTestId("home");
  expect(navElement).toBeInTheDocument();
  expect(homeElement).toBeInTheDocument();
});
