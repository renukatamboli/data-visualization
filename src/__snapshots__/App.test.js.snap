// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match snapshot 1`] = `
<div>
  <div
    class="App"
  >
    <div
      class="makeStyles-root-1"
      data-testid="nav"
    >
      <header
        class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
      >
        <div
          class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
        />
      </header>
    </div>
    <div
      data-testid="home"
    >
      <br />
      <input
        accept="image/*"
        data-testid="inputFile"
        id="fileUpload"
        multiple=""
        type="file"
      />
    </div>
  </div>
</div>
`;
